megamek/src/megamek/common/weapons/CapitalMissileBearingsOnlyHandler.java

```diff
@@ -43,41 +43,25 @@ public class CapitalMissileBearingsOnlyHandler extends AmmoBayWeaponHandler {
     boolean detRangeExtreme = (weapon.curMode().equals(Weapon.MODE_CAP_MISSILE_BEARING_EXT)
             || weapon.curMode().equals(Weapon.MODE_CAP_MISSILE_WAYPOINT_BEARING_EXT));
 
+    // Defined here so we can use it in multiple methods
+    Mounted bayWAmmo;
+    int range;
+    Coords targetCoords;
+
     /**
-     * This constructor may only be used for deserialization.
+     * This constructor can only be used for deserialization.
```

Comment:
Really don't get the beef with "may". It's a perfectly valid usage of the word.

Reply:
With further review it is completely valid, but can lead to misreads (as I did before making the naming change). Using "can" prevents those misreads.